particle_structs/test/CMakeLists.txt

```diff
@@ -6,6 +6,8 @@ function(make_test exename srcname)
   target_link_libraries(${exename} particleStructs)
 endfunction(make_test)
 
+make_test(smoke_test_particle smoke_test_particle.cpp)
```

Comment:
IIRC, the idea of a smoke test is that is enabled even when testing (`IS_TESTING`) is disabled.  

For the record, here is how smoke tests are defined in omegah: https://github.com/SCOREC/omega_h/blob/94aa915568abb01c169fc5b84bbbd12acede8935/src/CMakeLists.txt#L773-L797
and in the spack package (which is part of the e4s suite) doesn't enable testing (BUILD_TESTING is off by default):
https://github.com/spack/spack/blob/741652caa105edcc0414d6a8f0d94bd4ba20a736/var/spack/repos/builtin/packages/omega-h/package.py

Reply:
Thank you for the code example. I made the change.